bgpd/bgpd.c

```diff
@@ -7886,6 +7886,7 @@ void bgp_master_init(struct thread_master *master, const int buffer_size,
 	bm->socket_buffer = buffer_size;
 	bm->wait_for_fib = false;
 	bm->tcp_dscp = IPTOS_PREC_INTERNETCONTROL;
+	bm->inq_limit = BM_DEFAULT_INQ_LIMIT;
```

Comment:
Do we want to consider making this configurable per-peer? or would that just introduce unnecessary complexity?

Reply:
I considered it, but do you actually think there is use-case per-peer? Seem like to me if you have the memory and cpu to handle a large load you are going to just make it global.

Reply:
But also not sure if its worth dealing with it per-peer at this point. Can always add later if people ask for it.